examples/todomvc/test/todomvc/README.md

```diff
@@ -0,0 +1,156 @@
+# Testing with re-frame-test
+
+`re-frame-test` is a library which provides helper functions for testing re-frame projects
+
+## Philosopy 
```

Comment:
This doesn't need to be copied, as it will get out of date. Just refer the user to the root README?